timemachine/cpp/src/tibd_exchange_move.cu

```diff
@@ -254,15 +239,15 @@ void TIBDExchangeMove<RealType>::move(
             this->d_samples_.data,
             stream);
 
-        gpuErrchk(cudaEventSynchronize(host_copy_event_));
-        int inner_count = p_inner_count_.data[0];
-
-        // targeting_inner_vol == 1 indicates that we are targeting the inner volume, starting from the outer mols
```

Comment:
Looks like the logic for this was removed and not replicated elsewhere on the CPU, did all this pinned memory logic get cleaned up too? i.e. is the ternary decision now made on the GPU side?

Reply:
Yup, all the pinned code was cleaned up. The ternary decision was always made on the GPU side, just now we don't need to pull it to the host to run logsumexp.